src/test/java/peoplesoft/testutil/PersonUtil.java

```diff
@@ -37,6 +38,8 @@ public static String getPersonDetails(Person person) {
         sb.append(PREFIX_PHONE + person.getPhone().value + " ");
         sb.append(PREFIX_EMAIL + person.getEmail().value + " ");
         sb.append(PREFIX_ADDRESS + person.getAddress().value + " ");
+        // TODO might need more elegance
+        sb.append(PREFIX_RATE + person.getRate().getAmount().printFullValue() + " ");
```

Comment:
like the comment above, were you planning to implement toString() or .value?

Reply:
`printFullValue()`